pubspec.yaml

```diff
@@ -11,17 +11,17 @@ environment:
 
 dependencies:
   crypto: ^3.0.1
-  dio: ^4.0.0
+  dio: ^4.0.4
   flutter:
     sdk: flutter
-  json_serializable: ^4.1.4
+  json_serializable: ^6.1.4
```

Comment:
json_serializable should be in dev_dependencies not the normal dependencies,
because its not needed when compiling.

this will also prevent conflicts in the future.

Reply:
@agniswarmandal can you update this PR based on @clragon's comment?